seqr/utils/search/add_data_utils.py

```diff
@@ -116,17 +116,30 @@ def format_email(sample_summary, project_link, num_new_samples):
         )
 
 
-def prepare_data_loading_request(projects: list[Project], sample_type: str, dataset_type: str, genome_version: str,
-                                 data_path: str, user: User, pedigree_dir: str,  raise_pedigree_error: bool = False,
-                                 individual_ids: list[int] = None, skip_validation: bool = False):
-    project_guids = sorted([p.guid for p in projects])
-    variables = {
-        'projects_to_run': project_guids,
-        'callset_path': data_path,
-        'sample_type': sample_type,
+def format_loading_pipeline_variables(
+    projects: list[Project], genome_version: str, dataset_type: str, sample_type: str = None, **kwargs
+):
+    keyword_args = dict(**kwargs)
+    if sample_type:
+        keyword_args['sample_type'] = sample_type
```

Comment:
it would be a little neater not to cast `kwargs` to. dict - you could do that by doing 
```
variables = {
 'projects_to_run': ...,
  ...
  **kwargs,
}
if sample_type:
  variables['sample_type'] = sample_type
return variables
```